### ISSUE TITLE
Bug: File not output in respective directory

### ISSUE THREAD

**Issue by Jamesking56** [stidges/laravel-mix-mjml#4]
`webpack.mix.js`

```javascript
const mix = require('laravel-mix');
require('laravel-mix-mjml');

mix
  .mjml('src', 'public', {
    extension: '.html',
    beautify: true,
    minify: false
  })
;
```

Folder structure:

```
public
src
├── directory
    ├── 1.mjml
```

-----

Expected directory output after running `npm run dev`:

```
public
├── directory
    ├── 1.html
src
├── directory
    ├── 1.mjml
```

Actual directory output after running `npm run dev`:

```
public
src
├── directory
    ├── 1.mjml
    ├── 1publichtml     <---- wtf?
```

**Comment by Jamesking56** [comment]
Thanks @stidges :+1:

**Comment by stidges** [comment]
Published v2.0.1 just now - let me know if that resolved your issue!

**Comment by Jamesking56** [comment]
Just tested and it works perfectly :+1:
